### ISSUE TITLE
pipy / conda

### ISSUE THREAD

**Issue by lua-pele** [SJ001/AI-Feynman#2]
On Ubuntu18, conda environment with pytorch 1.3, python 3.6 I had to
apt install `gfortran` and `csh`
and on python `sortedcontainers, seaborn, sympy`
PiPy or conda version of this would be nice.

Also
cant the brute force steps not be parallelized with CUDA? 

Also2
on AMD Ryzen3900 32GRam, Titan RTX 24G system, your ai_feynman_example.py test  ran for over a day, then died without explanation (right after mystery_world_atan).  Never getting to the 5 layer MLP which I was curious about. FYI.

**Comment by SJ001** [comment]
Hello!

Thank you for the message and your interest in our code and I am sorry for the late reply. I added a requirements.txt file containing the required dependencies. We are planning to release a PiPy version of the code soon. Also, we are working on moving the Fortran part of the code to C++ and will allow the brute force code to be ran in parallel in the new code (thank you for the suggestions).
I am not totally sure where that error comes from, given that you didn't receive any error message. Could you please send me the full output of the code before the crush?

**Comment by lua-pele** [comment]
Thank you
I would have to run it again. Ill add some messages to your code to see whats happening and then open a new issue.

Sent with [ProtonMail](https://protonmail.com) Secure Email.

‐‐‐‐‐‐‐ Original Message ‐‐‐‐‐‐‐
